src/scripts/uploadAutoCompleteLocations.js

```diff
@@ -0,0 +1,357 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * The data that is being downloaded in this script is provided by
+ * GeoNames (https://www.geonames.org/). We are using the data to compile a set
+ * of US cities and states that match the needs of this project. Their work is
+ * licensed under a Creative Commons Attribution 4.0 License:
+ * https://creativecommons.org/licenses/by/4.0/.
+ *
+ * All database dumps and table definitions can be found here:
+ * https://download.geonames.org/export/dump/.
+ */
+
+import https from "https";
+import {
+  createWriteStream,
+  existsSync,
+  mkdirSync,
+  readFileSync,
+  rmSync,
+  writeFileSync,
+} from "fs";
+import { uploadToS3 } from "./s3.js";
+import Sentry from "@sentry/nextjs"
+import os from "os";
+import path from "path";
+import fs from "fs";
+import AdmZip from "adm-zip";
+
+const REMOTE_DATA_URL = "https://download.geonames.org/export/dump";
+const DATA_COUNTRY_CODE = "US";
+const LOCATIONS_DATA_FILE = "locationAutocompleteData.json";
+const FETCH_REMOTE_DATASETS = true;
+const CLEANUP_TMP_DATA_AFTER_FINISHED = true;
+const SENTRY_SLUG = "cron-create-location-autocomplete"
+
+Sentry.init({
+  dsn: process.env.SENTRY_DSN,
+  tracesSampleRate: 1.0,
+});
```

Comment:
Do we need to check if `process.env.SENTRY_DSN` is defined?
Or more specifically, skip Sentry logging if this is running in `APP_ENV:"local"`?